### PR TITLE
Doug b

### PR DESCRIPTION
**Pull request by notdb**

### CODE REVIEW

**Comment by notdb** [comment]
@samjeffcoat

File: assignments/objects.js [hunk 1]
```diff
@@ -19,6 +19,48 @@ const example = {
...
+const intern4 = {
+  "id": 4,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums: function(num1, num2) {
+    return num1*num2;
+  } 
+}
+
```
Comment (samjeffcoat) [inline assignments/objects.js:62]: Objects are written well.

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -4,22 +4,39 @@
...
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
-
+subtract = (param1, param2) => {
+    return param1 - param2;
+}
+subtract(1,2);
```
Comment (samjeffcoat) [inline assignments/function-conversion.js:41]: Function conversions are written correctly.

File: assignments/arrays.js [hunk 3]
```diff
@@ -64,31 +64,61 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+}
+
+
+for (let i=0; i<audiArray.length; i++) {
+    
+   BMWAndAudi.push(audiArray[i]);
+ }
+
+console.log(BMWAndAudi.length);
+console.log(JSON.stringify(BMWAndAudi));
```
Comment (samjeffcoat) [inline assignments/arrays.js:133]: Arrays are written correctly. Good job with the for loops.

**Review by samjeffcoat — APPROVED** [review]
JavaScript - I
## MVP
- [x ] Complete Object challenges without any errors
    - [x ] Challenge 1: Writing Objects
    - [x ] Challenge 2: Reading Object Data
    - [x ] Challenge 3: Object Methods
- [x ] Complete Array challenges without any errors
    - [ x] Challenge 1: Which car has an id of 33
    - [x ] Challenge 2: What is the make and model of the last car in the inventory?
    - [x ] Challenge 3: Sort all the car model names into alphabetical order
    - [x ] Challenge 4: Create a new array containing only the car years
    - [x ] Challenge 5: Populate the array oldCars (made before 2000) and log it's length
    - [x ] Challenge 6: Return an array that only contains BMW and Audi cars
- [ x] Arrow function syntax

## Stretch
- [x ] Object Challenges -- Nested Objects and the this keyword
- [x ] Use array methods on the array's assignment

Good:
Good job on getting to stretch.

Needs Improvement:
More frequent and descriptive commits

Grade:
2

**Pull request merged**